pandas/compat/__init__.py

```diff
@@ -139,7 +139,7 @@ def lfilter(*args, **kwargs):
     Hashable = collections.abc.Hashable
     Iterable = collections.abc.Iterable
     Mapping = collections.abc.Mapping
-    MutableMapping = collections.abc.MutableMapping
+    MutableMapping = collections.MutableMapping
```

Comment:
I think the one with `abc` included was actually correct? (CI is failing now due the warning this raises)